test/scripts/update.to.user.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
FYI I just added https://github.com/greasemonkey/greasemonkey/blob/master/test/scripts/script-server-update.py which might obsolete the need for this?